vault/audit_broker.go

```diff
@@ -0,0 +1,200 @@
+package vault
```

Comment:
Yeah splitting this out is a good idea. 👍

Reply:
Actually one issue: can you just explain what, if any, changes you made while splitting this out? It's otherwise hard to tell without comparing a huge diff line by line.

Reply:
The method signatures for `LogRequest` and `LogResponse` got changed, and any of the old param references inside those got update to use `in *audit.LogInput` instead.